dev_tools/docs/build_api_docs.py

```diff
@@ -86,5 +92,25 @@ def main(unused_argv):
     doc_generator.build(output_dir=FLAGS.output_dir)
 
 
+def generate_cirq_google():
+    doc_generator = generate_lib.DocGenerator(
+        root_title="Cirq-google",
+        py_modules=[("cirq_google", cirq_google)],
+        base_dir=os.path.dirname(cirq_google.__file__),
+        code_url_prefix=FLAGS.code_url_prefix + "/cirq-google/cirq_google",
+        search_hints=FLAGS.search_hints,
+        site_path=FLAGS.site_path,
+        callbacks=[public_api.local_definitions_filter, filter_unwanted_inherited_methods],
+        private_map={
+            # Opt to not build docs for these paths for now since they error.
+            "cirq_google.engine.client.quantum.QuantumEngineServiceClient": ["enums"],
+            "cirq_google.engine.client.quantum_v1alpha1.QuantumEngineServiceClient": ["enums"],
+            "cirq_google.api": ["v1"],
+        },
+        extra_docs=_doc.RECORDED_CONST_DOCS,
```

Comment:
Does `cirq_google` need this? It lives in `cirq-core`, and seems to be `cirq`-specific.

Reply:
Yes we need it here, this helps resolving docstrings for cirq-google public constants too e.g here: https://github.com/quantumlib/Cirq/blob/master/cirq-google/cirq_google/gate_sets.py#L70